src/client/helpers/postHelpers.js

```diff
@@ -17,4 +19,25 @@ export function dropCategory(url) {
   return url.replace(categoryRegex, '');
 }
 
+/**
+ * Gets app data from a post.
+ * Only Returns app info from apps whitelisted in apps.json
+ * @param post
+ * @returns {*}
+ */
+export function getAppData(post) {
+  let appName;
+  let version;
+  try {
+    const app = jsonParse(post.json_metadata).app.split('/');
```

Comment:
this will fail if `.app` is undefined, and this will mostly be the case, since so many apps on steem blockchain are popping up and they might not include the app meta data, you should probably do `_.split()` here.

Reply:
I think in this case it would be ok, because of the try/catch.  Ill refactor using lodash as you suggested and remove the try/catch. Should make cleaner without the try/catch.